examples/todoapp/lib/base/data_sources/local/todo_local_data_source.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:realm/realm.dart';
+import '../../models/todo_model.dart';
+
+class TodoLocalDataSource {
+  final Realm realm;
+  TodoLocalDataSource(this.realm);
+
+// .query(r'action != $0 SORT(createdAt ASC)', ['delete'])
+  Stream<List<$TodoModel>> allTodosStream() => realm
```

Comment:
I think that there is no need to add the word stream as part of the method name 

- fetchTodoList()
- todoTodoList()